osidb/serializer.py

```diff
@@ -502,6 +516,66 @@ class Meta:
         abstract = True
 
 
+class NewFlawCommentMixinSerializer(serializers.ModelSerializer):
+    """
+    serializer mixin class implementing special handling of pending new comments
+    """
+
+    def to_representation(self, instance):
+        """
+        Perform normal to_representation, but first delete stale pending
+        comments.
+        """
+        return super().to_representation(instance)
```

Comment:
Based on our discussion, something is probably missing here.

Reply:
Good catch. This code will go away based on Adrian's feedback, but I fixed it nonetheless, so that the current state is consistent at least.